src/config/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dataVaultUrl": "https://identity.staging.rifcomputing.net"
```

Comment:
Don't we need to add the `v0`?

Reply:
Thanks for catching this. Yes, I have updated it.